christfellowshipchurch/web-app-v2
# Changed FAQ Scrolling

### About
* Removed Scroll Position using breakpoints and pixels and changed it to scrolling to the top of the FAQ section using id and scroll to element so that when elements above are added or removed in any of the locations pages we do not have to manually change the scroll location.

### Test Instructions
1. Go to the deployment link and go to any location - for example [/locations/palm-beach-gardens](https://www.christfellowship.church/locations/palm-beach-gardens)
2. Go to the live website and the same location - [/locations/palm-beach-gardens](https://web-app-v2-git-faq-scroll-fix-2-christ-fellowship-church.vercel.app/locations/palm-beach-gardens)
3. Compare - When clicking on "See Less" or "See More" the webpage should scroll to the appropriate location.
4. Right click - Inspect - Try clicking on "See Less" or "See More" on different devices.
5. Follow the above steps on a different location.

### Screenshots
|Old|New|
|--|--|
|<img width="1440" alt="Screenshot 2023-06-06 at 12 30 26 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/30ce667d-aba1-45fd-8eb0-4a6dbc274711">|<img width="1440" alt="Screenshot 2023-06-06 at 12 41 59 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/edf0cf32-c335-482c-95b9-f16042b44a11">|

### Closes Tickets
[CFDP-2595](https://christfellowshipchurch.atlassian.net/browse/CFDP-2595)


[CFDP-2595]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2595?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
